include/exadg/fluid_structure_interaction/single_field_solvers/fluid.h

```diff
@@ -148,10 +148,9 @@ SolverFluid<dim, Number>::setup(std::shared_ptr<FluidFSI::ApplicationBase<dim, N
   // initialize time_integrator
   helpers_ale = std::make_shared<HelpersALE<Number>>();
 
+  bool print_solver_info = is_test ? false : time_integrator->print_solver_info();
```

Comment:
I am not sure if this works as intended because `time_integrator->print_solver_info()` might be `true/false` depending on the time step (say solver info is going to be printed every 10th time step). However, the variable `print_solver_info` will then take the same value throughout a whole simulation, doesn't it?

Reply:
I moved it inside now, this is work in progress :)